ably/realtime_client.go

```diff
@@ -32,11 +32,22 @@ func NewRealtime(options ...ClientOption) (*Realtime, error) {
 		c.onChannelMsg,
 		c.onReconnected,
 		c.onReconnectionFailed,
-	})
+	}, c)
 	conn.internalEmitter.OnAll(func(change ConnectionStateChange) {
 		c.Channels.broadcastConnStateChange(change)
 	})
 	c.Connection = conn
+
+	// RTN16
+	if !empty(c.opts().Recover) {
+		recoverKeyContext, err := DecodeRecoveryKey(c.opts().Recover)
+		if err != nil {
+			c.log().Errorf("Error decoding recover with error %v", err)
```

Comment:
Should the error be returned? If not, then I think it's worth commenting that it is intentionally ignored, otherwise it looks like a bug.

Reply:
Actually it's intentionally ignored! 
Will update logging message for the same

Reply:
updated

Reply:
Nice, thanks